packages/demos/src/module-loader-federation/webpack-config.js

```diff
@@ -0,0 +1,98 @@
+// @ts-check
+const ModuleFederationPlugin = require('webpack').container
+  .ModuleFederationPlugin;
+
+const path = require('path');
+//const webpackBaseConfig = require('../webpack-base-config');
+
+const configs = [
+  {
+    entry: path.join(__dirname, './feature-app.tsx'),
+    mode: 'development',
+    output: {
+      //filename: 'feature-app.umd.js',
+      //libraryTarget: 'umd',
+      publicPath: '/',
+    },
+    resolve: {
+      extensions: ['.ts', '.tsx', '.js'],
+    },
+    module: {
+      rules: [
+        {
+          test: /\.tsx?$/,
+          loader: 'babel-loader',
+          exclude: /node_modules/,
+          options: {
+            presets: ['@babel/preset-react', '@babel/preset-typescript'],
+          },
+        },
+        {
+          test: /\.css$/,
+          use: ['style-loader', 'css-loader'],
+        },
+        {
+          test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+          use: 'url-loader',
+        },
+      ],
+    },
+    plugins: [
+      new ModuleFederationPlugin({
+        filename: 'remoteEntry.js',
+        name: 'remoteEntry',
+        exposes: {
+          './featureAppDefinition':
```

Comment:
each expos will result in a new chunk whose name is derived from the path.

Reply:
now using a unique global `featurehubGlobal` and exposing the standard `./featureAppDefinition`